content/docs/how-we-work/ci-cd.md

```diff
@@ -14,6 +14,12 @@ Like most errors we see when dealing with computers, errors reported by CI are n
 
 GitHub Actions tout themselves as an automation platform, not just CI/CD. In our projects, these are mostly used for CI/CD tasks. It is useful to know how you could run other forms of automation to manage a project on GitHub but we won't focus on that here.
 
+To set up GitHub Actions for your project, follow these steps:
+
+Create .github/workflows directory in your project root. You can add your workflow files here. Refer to our templates and GitHub Actions Manual for more details on what the workflow files look like.
```

Comment:
I see you have replaced the steps with a single line but you have not changed the previous line. There are no steps anymore and this doesn't make sense.

Reply:
Please rephrase this entire block so that it flows nicely.